swagger.yaml

```diff
@@ -4270,6 +4270,8 @@ definitions:
   disk:
     type: object
     properties:
+      id:
```

Comment:
Neither the commit message nor the JIRA describes what this new field is or why it's needed.

Reply:
Added the info to the design document, PR and commit description https://docs.google.com/document/d/1L1_y3NDvx_NTNKK2VZrc9IFabnrd37P_jpiyS9gjAnw/edit